### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [S73S/CapstoneColmar#1]
https://github.com/S73S/CapstoneColmar/blob/5ad8e7723aa7f959c13506bca0babef9dbbc5910/index.html#L62-L63

All images should have descriptive alt text. The visually impaired rely on alt text through the use of screen readers on their computers and mobile devices.
